identity-service/src/config.js

```diff
@@ -617,6 +617,12 @@ const config = convict({
     format: String,
     env: 'discoveryProviderWhitelist',
     default: ''
+  },
+  clusterForkProcessCount: {
+    doc: 'The number of express server processes to initialize in the this app "cluster"',
+    format: Number,
+    env: 'clusterForkProcessCount',
+    default: 1
```

Comment:
this has to be 1 for the time being because eth and poa relays lock wallets in memory. we would need to move that logic to redis to parallelize horizontally